packages/dds/tree/src/test/domains/json/jsonCursor.spec.ts

```diff
@@ -75,6 +80,13 @@ describe("JsonCursor", () => {
             assert.deepEqual([...new JsonCursor(false).keys], []);
             assert.deepEqual([...new JsonCursor(true).keys], []);
         });
+
+        it("null", () => {
+            // 'ITreeCursor.keys' enumerates the keys of the current node, which in the JSON domain
+            // includes a node with value 'null'.  This behavior differs from 'Object.keys(null)',
+            // which throws a TypeError.
+            assert.deepEqual([...new JsonCursor(null).keys], []);
```

Comment:
@CraigMacomber - I expanded a bit to capture the intentional difference with the 'Object.keys()' behavior.